src/TorchSharp/NN/Module.cs

```diff
@@ -418,30 +412,35 @@ public virtual IEnumerable<(string name, Module module)> named_modules()
                 /// </summary>
                 /// <param name="source">A dict containing parameters and persistent buffers.</param>
                 /// <param name="strict">Whether to strictly enforce that the keys in state_dict match the keys returned by this module’s state_dict() function.</param>
+                /// <param name="skipped">A list of keys not to consider when loading the dictionary.</param>
                 /// <returns></returns>
-                public virtual (IList<string> missing_keys, IList<string> unexpected_keyes) load_state_dict(Dictionary<string, Tensor> source, bool strict = true)
+                public virtual (IList<string> missing_keys, IList<string> unexpected_keyes) load_state_dict(Dictionary<string, Tensor> source, bool strict = true, IList<string> skipped = null)
                 {
                     List<string> missing = new List<string>();
                     List<string> unexpected = new List<string>();
+                    if (skipped is null) skipped = new List<string>();
```

Comment:
>if (skipped is null) skipped = new List<string>(); [](http://example.com/codeflow?start=20&length=50)

nit: 

```C#
skipped ??= new List<string>();
```

Reply:
All these new-fangled C# ways of dealing with null... :-)

Thanks!

Reply:
Actually would be better to have something like:

```
privare static readonly s_emptyList = new List<string>();

.... C#
skipped ??= s_emptyList;
```

Reply:
Sure. Array.Empty, perhaps?

Reply:
Even better as arrays implement IList.